### ISSUE TITLE
Trouble using Logic3 TopDrive Reactor

### ISSUE THREAD

**Issue by Warpshock** [sonik-br/RetroZordAdapter#5]
I am interested in the Reflex Adapt and using it with the Logic3 TopDrive Reactor, but figured I would try the RetroZordAdapter first. I assembled the RetroZordAdapter and was able to get it to work with the MadCatz Handheld Racing Wheel. 

When I connected the TopDrive Reactor, no signals are sent to the Windows gamepad tester. I have two TopDrive Reactors and have tried each of the TopDrive's mode settings, replugging the RetroZord each time, and nothing changed. I have tried with and without the 9V adapter.

Both controllers work with the Logic3 PSX to USB adapter, but without FFB (I don't think it supports FFB). Any thoughts on why the RetroZordAdapter doesn't like my Reactors?

**Comment by sonik-br** [comment]
Hi.

I do have a TopDrive Reactor too.
It's jogcon mode does not work exactly like a namco jogcon and the adapter does not support it when in jogcon mode.
Negcon mode should work. I can test it on mine and report back.

Btw the psx mode firmware is missing some settings.
Put this on the top of the main sketch file, flash the arduino and try the controllers again.
```
#define NEGCON_SUPPORT
#define JOGCON_SUPPORT
```

**Comment by sonik-br** [comment]
Tested with mine and indeed it does not work as expected.
But if you reset the arduino with the TopDrive Reactor connected, it will be detected after the reset process and will work (in negcon mode).
Let me know if this workaround also works for you.

**Comment by Warpshock** [comment]
I added the defines and reworked the power. Occasionally I am able to get button presses in dualshock and negcon modes, but I am not getting any analog values. 

Any ideas what would cause this?

> Tested with mine and indeed it does not work as expected. But if you reset the arduino with the TopDrive Reactor connected, it will be detected after the reset process and will work (in negcon mode). Let me know if this workaround also works for you.

I added the defines and reworked the power. Occasionally I am able to get button presses in dualshock and negcon modes, but I am not getting any analog values. 

Any ideas what would cause this?
